build-scripts/rcm-guest/publish-oc-binary.sh

```diff
@@ -9,22 +9,20 @@ rpm_name() {
 
 extract() {
     local arch rpm
-    mkdir macosx windows
-    for arch in x86_64 ${ARCH}; do
+    mkdir macosx windows x86_64 ${ARCH[@]}
+    for arch in x86_64 ${ARCH[@]}; do
         rpm=$(echo "$(rpm_name "${arch}")"*)
-        if [[ "${arch}" != x86_64 && ! -e "${rpm}" ]]; then continue; fi
-        mkdir "${arch}"
```

Comment:
This is a semantic change (although it might be irrelevant).

The original code only creates directories for architectures that have rpms.

Reply:
@bbguimaraes  Fixed in https://github.com/openshift/aos-cd-jobs/pull/1450/commits/d351158fd76c00e5ceb5ae5fa2ac1ad8a99b6cb7